I can't Build zipkin-Server it's always build failure

I use the command 
```
git clone https://github.com/openzipkin/zipkin
cd zipkin
./mvnw -DskipTests --also-make -pl zipkin-server clean install
```
then
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Zipkin (Parent) 2.13.0-SNAPSHOT:
[INFO]
[INFO] Zipkin (Parent) .................................... SUCCESS [02:30 min]
[INFO] Zipkin Core Library ................................ SUCCESS [ 46.815 s]
[INFO] Zipkin Interop Tests ............................... SUCCESS [  9.079 s]
[INFO] Zipkin UI .......................................... SUCCESS [01:14 min]
[INFO] Zipkin Lens ........................................ SUCCESS [01:10 min]
[INFO] Collector .......................................... SUCCESS [  0.612 s]
[INFO] Collector: Core Library ............................ SUCCESS [  1.685 s]
[INFO] Storage ............................................ SUCCESS [  0.563 s]
[INFO] Storage: Cassandra (Zipkin V2) ..................... SUCCESS [01:03 min]
[INFO] Storage: Cassandra (v1) ............................ SUCCESS [  5.553 s]
[INFO] Storage: Elasticsearch (V2) ........................ SUCCESS [ 16.455 s]
[INFO] Storage: MySQL (v1) ................................ SUCCESS [ 13.840 s]
[INFO] Collector: Kafka 0.10+ ............................. SUCCESS [01:04 min]
[INFO] Collector: RabbitMQ ................................ SUCCESS [ 19.452 s]
[INFO] Zipkin Server ...................................... FAILURE [04:53 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  14:00 min
[INFO] Finished at: 2019-04-29T10:20:40+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.squareup.wire:wire-maven-plugin:3.0.0-alpha01:generate-sources (default) on project zipkin-server: Wire Plugin: Failure compiling proto sources.: google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field file (google/protobuf/descriptor.proto at 57:3)
[ERROR]   in message google.protobuf.FileDescriptorSet (google/protobuf/descriptor.proto at 56:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field message_type (google/protobuf/descriptor.proto at 74:3)
[ERROR]   in message google.protobuf.FileDescriptorProto (google/protobuf/descriptor.proto at 61:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field enum_type (google/protobuf/descriptor.proto at 75:3)
[ERROR]   in message google.protobuf.FileDescriptorProto (google/protobuf/descriptor.proto at 61:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field service (google/protobuf/descriptor.proto at 76:3)
[ERROR]   in message google.protobuf.FileDescriptorProto (google/protobuf/descriptor.proto at 61:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field extension (google/protobuf/descriptor.proto at 77:3)
[ERROR]   in message google.protobuf.FileDescriptorProto (google/protobuf/descriptor.proto at 61:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field options (google/protobuf/descriptor.proto at 79:3)
[ERROR]   in message google.protobuf.FileDescriptorProto (google/protobuf/descriptor.proto at 61:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field source_code_info (google/protobuf/descriptor.proto at 85:3)
[ERROR]   in message google.protobuf.FileDescriptorProto (google/protobuf/descriptor.proto at 61:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field field (google/protobuf/descriptor.proto at 96:3)
[ERROR]   in message google.protobuf.DescriptorProto (google/protobuf/descriptor.proto at 93:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field extension (google/protobuf/descriptor.proto at 97:3)
[ERROR]   in message google.protobuf.DescriptorProto (google/protobuf/descriptor.proto at 93:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field nested_type (google/protobuf/descriptor.proto at 99:3)
[ERROR]   in message google.protobuf.DescriptorProto (google/protobuf/descriptor.proto at 93:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field enum_type (google/protobuf/descriptor.proto at 100:3)
[ERROR]   in message google.protobuf.DescriptorProto (google/protobuf/descriptor.proto at 93:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field extension_range (google/protobuf/descriptor.proto at 108:3)
[ERROR]   in message google.protobuf.DescriptorProto (google/protobuf/descriptor.proto at 93:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field oneof_decl (google/protobuf/descriptor.proto at 110:3)
[ERROR]   in message google.protobuf.DescriptorProto (google/protobuf/descriptor.proto at 93:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field options (google/protobuf/descriptor.proto at 112:3)
[ERROR]   in message google.protobuf.DescriptorProto (google/protobuf/descriptor.proto at 93:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field reserved_range (google/protobuf/descriptor.proto at 121:3)
[ERROR]   in message google.protobuf.DescriptorProto (google/protobuf/descriptor.proto at 93:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field options (google/protobuf/descriptor.proto at 106:5)
[ERROR]   in message google.protobuf.DescriptorProto.ExtensionRange (google/protobuf/descriptor.proto at 102:3)
[ERROR]   in message google.protobuf.DescriptorProto (google/protobuf/descriptor.proto at 93:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field uninterpreted_option (google/protobuf/descriptor.proto at 129:3)
[ERROR]   in message google.protobuf.ExtensionRangeOptions (google/protobuf/descriptor.proto at 127:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field label (google/protobuf/descriptor.proto at 179:3)
[ERROR]   in message google.protobuf.FieldDescriptorProto (google/protobuf/descriptor.proto at 136:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field type (google/protobuf/descriptor.proto at 183:3)
[ERROR]   in message google.protobuf.FieldDescriptorProto (google/protobuf/descriptor.proto at 136:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field options (google/protobuf/descriptor.proto at 213:3)
[ERROR]   in message google.protobuf.FieldDescriptorProto (google/protobuf/descriptor.proto at 136:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field options (google/protobuf/descriptor.proto at 219:3)
[ERROR]   in message google.protobuf.OneofDescriptorProto (google/protobuf/descriptor.proto at 217:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field value (google/protobuf/descriptor.proto at 226:3)
[ERROR]   in message google.protobuf.EnumDescriptorProto (google/protobuf/descriptor.proto at 223:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field options (google/protobuf/descriptor.proto at 228:3)
[ERROR]   in message google.protobuf.EnumDescriptorProto (google/protobuf/descriptor.proto at 223:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field reserved_range (google/protobuf/descriptor.proto at 244:3)
[ERROR]   in message google.protobuf.EnumDescriptorProto (google/protobuf/descriptor.proto at 223:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field options (google/protobuf/descriptor.proto at 256:3)
[ERROR]   in message google.protobuf.EnumValueDescriptorProto (google/protobuf/descriptor.proto at 252:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field method (google/protobuf/descriptor.proto at 262:3)
[ERROR]   in message google.protobuf.ServiceDescriptorProto (google/protobuf/descriptor.proto at 260:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field options (google/protobuf/descriptor.proto at 264:3)
[ERROR]   in message google.protobuf.ServiceDescriptorProto (google/protobuf/descriptor.proto at 260:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field options (google/protobuf/descriptor.proto at 276:3)
[ERROR]   in message google.protobuf.MethodDescriptorProto (google/protobuf/descriptor.proto at 268:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field optimize_for (google/protobuf/descriptor.proto at 361:3)
[ERROR]   in message google.protobuf.FileOptions (google/protobuf/descriptor.proto at 318:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field uninterpreted_option (google/protobuf/descriptor.proto at 422:3)
[ERROR]   in message google.protobuf.FileOptions (google/protobuf/descriptor.proto at 318:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field uninterpreted_option (google/protobuf/descriptor.proto at 490:3)
[ERROR]   in message google.protobuf.MessageOptions (google/protobuf/descriptor.proto at 431:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field ctype (google/protobuf/descriptor.proto at 501:3)
[ERROR]   in message google.protobuf.FieldOptions (google/protobuf/descriptor.proto at 496:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field jstype (google/protobuf/descriptor.proto at 528:3)
[ERROR]   in message google.protobuf.FieldOptions (google/protobuf/descriptor.proto at 496:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field uninterpreted_option (google/protobuf/descriptor.proto at 581:3)
[ERROR]   in message google.protobuf.FieldOptions (google/protobuf/descriptor.proto at 496:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field uninterpreted_option (google/protobuf/descriptor.proto at 591:3)
[ERROR]   in message google.protobuf.OneofOptions (google/protobuf/descriptor.proto at 589:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field uninterpreted_option (google/protobuf/descriptor.proto at 612:3)
[ERROR]   in message google.protobuf.EnumOptions (google/protobuf/descriptor.proto at 597:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field uninterpreted_option (google/protobuf/descriptor.proto at 626:3)
[ERROR]   in message google.protobuf.EnumValueOptions (google/protobuf/descriptor.proto at 618:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field uninterpreted_option (google/protobuf/descriptor.proto at 646:3)
[ERROR]   in message google.protobuf.ServiceOptions (google/protobuf/descriptor.proto at 632:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field idempotency_level (google/protobuf/descriptor.proto at 673:3)
[ERROR]   in message google.protobuf.MethodOptions (google/protobuf/descriptor.proto at 652:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field uninterpreted_option (google/protobuf/descriptor.proto at 677:3)
[ERROR]   in message google.protobuf.MethodOptions (google/protobuf/descriptor.proto at 652:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field name (google/protobuf/descriptor.proto at 700:3)
[ERROR]   in message google.protobuf.UninterpretedOption (google/protobuf/descriptor.proto at 690:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field location (google/protobuf/descriptor.proto at 761:3)
[ERROR]   in message google.protobuf.SourceCodeInfo (google/protobuf/descriptor.proto at 717:1)
[ERROR] google/protobuf/descriptor.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field annotation (google/protobuf/descriptor.proto at 854:3)
[ERROR]   in message google.protobuf.GeneratedCodeInfo (google/protobuf/descriptor.proto at 851:1)
[ERROR] google\protobuf\compiler\plugin.proto needs to import google\protobuf\descriptor.proto
[ERROR]   for field proto_file (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\compiler\plugin.proto at 90:3)
[ERROR]   in message google.protobuf.compiler.CodeGeneratorRequest (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\compiler\plugin.proto at 67:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field fields (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 51:3)
[ERROR]   in message google.protobuf.Type (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 47:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field options (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 55:3)
[ERROR]   in message google.protobuf.Type (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 47:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field syntax (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 59:3)
[ERROR]   in message google.protobuf.Type (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 47:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field kind (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 119:3)
[ERROR]   in message google.protobuf.Field (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 63:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field cardinality (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 121:3)
[ERROR]   in message google.protobuf.Field (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 63:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field options (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 135:3)
[ERROR]   in message google.protobuf.Field (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 63:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field enumvalue (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 147:3)
[ERROR]   in message google.protobuf.Enum (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 143:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field options (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 149:3)
[ERROR]   in message google.protobuf.Enum (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 143:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field syntax (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 153:3)
[ERROR]   in message google.protobuf.Enum (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 143:1)
[ERROR] google\protobuf\type.proto needs to import google/protobuf/type.proto
[ERROR]   for field options (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 163:3)
[ERROR]   in message google.protobuf.EnumValue (E:\workspace\zipkin\zipkin\zipkin-server\target\test\proto\google\protobuf\type.proto at 157:1)
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :zipkin-server

```

--- a/benchmarks/pom.xml
+++ b/benchmarks/pom.xml
@@ -80,21 +80,8 @@
       <version>${wire.version}</version>
     </dependency>
     <dependency>
-      <groupId>io.zipkin.proto3</groupId>
+      <groupId>org.apache.zipkin.proto3</groupId>
       <artifactId>zipkin-proto3</artifactId>
-    </dependency>
-    <!-- TODO: This only resolves google empty. Remove this once using org.apache.zipkin.proto3 -->
-    <dependency>
-      <groupId>com.google.protobuf</groupId>
-      <artifactId>protobuf-java</artifactId>
-      <version>3.7.1</version>
-      <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>*</groupId>
-          <artifactId>*</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
   </dependencies>
 

--- a/pom.xml
+++ b/pom.xml
@@ -382,9 +382,9 @@
       </dependency>
 
       <dependency>
-        <groupId>io.zipkin.proto3</groupId>
+        <groupId>org.apache.zipkin.proto3</groupId>
         <artifactId>zipkin-proto3</artifactId>
-        <version>0.2.0</version>
+        <version>0.2.1</version>
       </dependency>
       <dependency>
         <groupId>com.squareup.wire</groupId>
@@ -557,8 +557,7 @@
                 <goal>unpack-dependencies</goal>
               </goals>
               <configuration>
-                <!-- TODO: Remove protobuf-java once using org.apache.zipkin.proto3 -->
-                <includeArtifactIds>protobuf-java,zipkin-proto3</includeArtifactIds>
+                <includeArtifactIds>zipkin-proto3</includeArtifactIds>
                 <includes>**/*.proto</includes>
                 <outputDirectory>${unpack-proto.directory}</outputDirectory>
               </configuration>
@@ -578,8 +577,6 @@
               <configuration>
                 <protoSourceDirectory>${unpack-proto.directory}</protoSourceDirectory>
                 <includes>
-                  <!-- TODO: Remove google.protobuf.Empty once using org.apache.zipkin.proto3 -->
-                  <include>google.protobuf.Empty</include>
                   <include>zipkin.proto3.*</include>
                 </includes>
               </configuration>

--- a/zipkin-server/pom.xml
+++ b/zipkin-server/pom.xml
@@ -133,22 +133,9 @@
 
     <!-- to test the experimental grpc endpoint with the square/wire library -->
     <dependency>
-      <groupId>io.zipkin.proto3</groupId>
+      <groupId>org.apache.zipkin.proto3</groupId>
       <artifactId>zipkin-proto3</artifactId>
       <scope>test</scope>
-    </dependency>
-    <!-- TODO: This only resolves google empty. Remove this once using org.apache.zipkin.proto3 -->
-    <dependency>
-      <groupId>com.google.protobuf</groupId>
-      <artifactId>protobuf-java</artifactId>
-      <version>3.7.1</version>
-      <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>*</groupId>
-          <artifactId>*</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
     <dependency>
       <groupId>com.squareup.wire</groupId>

--- a/zipkin-server/src/test/kotlin/zipkin2/server/internal/ITZipkinGrpcCollector.kt
+++ b/zipkin-server/src/test/kotlin/zipkin2/server/internal/ITZipkinGrpcCollector.kt
@@ -17,7 +17,6 @@
 
 package zipkin2.server.internal
 
-import com.google.protobuf.Empty
 import com.linecorp.armeria.server.Server
 import com.squareup.wire.GrpcClient
 import com.squareup.wire.Service
@@ -38,6 +37,7 @@ import zipkin2.TestObjects
 import zipkin2.codec.SpanBytesDecoder
 import zipkin2.codec.SpanBytesEncoder
 import zipkin2.proto3.ListOfSpans
+import zipkin2.proto3.ReportResponse
 import zipkin2.storage.InMemoryStorage
 
 /** This tests that we accept messages constructed by other clients. */
@@ -58,9 +58,9 @@ class ITZipkinGrpcCollector {
     @WireRpc(
       path = "/zipkin.proto3.SpanService/Report",
       requestAdapter = "zipkin2.proto3.ListOfSpans#ADAPTER",
-      responseAdapter = "com.google.protobuf.Empty#ADAPTER"
+      responseAdapter = "zipkin2.proto3.ReportResponse#ADAPTER"
     )
-    suspend fun Report(request: ListOfSpans): Empty
+    suspend fun Report(request: ListOfSpans): ReportResponse
   }
 
   @Before fun sanityCheckCodecCompatible() {

